Fixed ad not being muted

albarozzz opened the pull request:
#6

albarozzz committed:
Changed ad condition

albarozzz committed:
Minor change

Changed 'ad' to 'ad/' so there are no conflicts with a track id?

KivalM commented:
Ideally, we would like this to keep working with all versions of the client.
My fear is that the trackid change was isolated, so we would still encounter the old format some of the time.

If you can add this as a third condition to your PR rather than replacing the existing one, that would be great.

Something like `response.starts_with("spotify:ad:") ||  response.starts_with("/com/spotify/ad/")||response.is_empty(); `

albarozzz committed:
Changed ad condition, again

to 'response.starts_with("spotify:ad:") || response.starts_with("/com/spotify/ad/")||response.is_empty() '

albarozzz commented:
Done!

KivalM commented:
Looks to be all good here. 
Thanks for contributing
